tests/test_utils.py

```diff
@@ -313,3 +313,14 @@ def test_check_status_code_for_missing_status_code(capsys, repo_url):
 
     captured = capsys.readouterr()
     assert captured.out == "NOTE: Try --verbose to see response content.\n"
+
+
+def test_get_file_size_unit_determination():
+    """Test if get_file_size returns a non zero value in KB for a file < .01 MB."""
+    test_file = helpers.SDIST_FIXTURE
+
+    file_size, size_unit = utils.get_file_size(test_file)
+    print(file_size)
+    print(size_unit)
+    assert file_size != 0
+    assert size_unit == "KB"
```

Comment:
Nice to see a unit test for `get_file_size`, but could use some work. Specifically, removing the `print` statements, and testing with different file sizes (ideally using [pytest parameterization](https://docs.pytest.org/en/stable/parametrize.html).

Also, it looks like you created this branch before we merged a rewrite of the test docstrings. We're not using words like "Test", opting instead to use imperative mood to describe the expected behavior. See https://github.com/pypa/twine/commit/194eccdde19a2064c3e4a22e7d9364bf9fcf7992 for examples.